nixnet/_session/frames.py

```diff
@@ -222,7 +222,7 @@ def write(
             frames,
             timeout=10):
         # type: (typing.Iterable[types.Frame], float) -> None
-        """Write raw CAN frame data.
+        """Write raw CAN or LIN frame data.
```

Comment:
Most of these comments are out of date, sorry for that.

Please change them to be "Read frames" (no reference to "raw", "CAN", or "LIN").